docs/operators/droplet-quickstart.md

```diff
@@ -0,0 +1,240 @@
+
+## 4. Select the OS distribution
+
+Run the following command to get the list of distributions -
+
+```bash
+doctl compute image list-distribution
+```
+
+You should get an output similar to this -
+
+```bash
+ID           Name                  Type        Distribution    Slug                   Public    Min Disk
+78547182     1.5.8 x64             snapshot    RancherOS       rancheros              true      15
+106433672    7 x64                 snapshot    CentOS          centos-7-x64           true      9
+106434098    9 Stream x64          snapshot    CentOS          centos-stream-9-x64    true      10
+106434191    8 Stream x64          snapshot    CentOS          centos-stream-8-x64    true      10
+...
+```
+
+Choose the distribution you are most comfortable with, and then run the following command
+
+```bash
+export DROPLET_IMAGE=<slug>
+```
+
+For example, if you chose Debian 11 x64 -
+
+```bash
+export DROPLET_IMAGE=debian-11-x64
+```
+
+## 5. Select the size of the Droplet
+
+Run the following command to get the list of Droplet sizes for the previously selected region -
+
+```bash
+doctl compute size list
+```
+
+You should get an output similar to this -
+```bash
+Slug                  Description                   Memory    VCPUs    Disk    Price Monthly    Price Hourly
+s-1vcpu-512mb-10gb    Basic                         512       1        10      4.00             0.005950
+s-1vcpu-1gb           Basic                         1024      1        25      6.00             0.008930
+s-1vcpu-1gb-amd       Basic AMD                     1024      1        25      7.00             0.010420
+s-1vcpu-1gb-intel     Basic Intel                   1024      1        25      7.00             0.010420
+s-1vcpu-2gb           Basic                         2048      1        50      12.00            0.017860
+s-1vcpu-2gb-amd       Basic AMD                     2048      1        50      14.00            0.020830
+s-1vcpu-2gb-intel     Basic Intel                   2048      1        50      14.00            0.020830
+s-2vcpu-2gb           Legacy Basic                  2048      2        60      18.00            0.026790
+...
+```
+
+> Note: To compile the nwaku binary, a minimum of 2GB of RAM is required. You may choose a smaller Droplet, however, you would have to supply the binary in an alternate manner, i.e via the official release on Github, or compiling it on another machine and copying it over. Currently, we only supply binaries for macOS and Ubuntu.
+
+Choose the Droplet size that you are most comfortable with, and then run the following command -
+
+```bash
+export DROPLET_SIZE=<slug>
+```
+
+For example, `s-1vcpu-2gb` is more than capable to handle the protocols we mentioned above -
+
+```bash
+export DROPLET_SIZE=s-1vcpu-2gb
+```
+
+## 6. Create the Droplet
+
+Run the following command to create the droplet -
+
+```bash
+export DROPLET_NAME=<your-droplet-name>
```

Comment:
A
> Note

below to mention that droplet names can only contain valid hostname characters, i.e. `a-z, A-Z, 0-9, . and -`

Reply:
Addressed in [3febd72](https://github.com/status-im/nwaku/pull/1133/commits/3febd72c0923543883064290187a999d3f291c4e)